node/packages/parser/src/ethereum/zrx.ts

```diff
@@ -1,11 +1,18 @@
 import { Tx } from '@shapeshiftoss/blockbook'
-import { Dex, Tx as ParseTx, TradeType } from '../types'
+import { Dex, TradeType, Tx as ParseTx } from '../types'
+import { txMatchesContract } from './helpers'
 
 export const PROXY_CONTRACT = '0xDef1C0ded9bec7F1a1670819833240f027b25EfF'
 
 export class Parser {
+  isSupportedTransaction(tx: Tx) {
```

Comment:
You like the sounds of `isParsable` better?

Reply:
Can see this being a required part of the interface

Reply:
Definitely, `isParsable` better conveys its relationship with `parse()`.

Renamed in https://github.com/shapeshift/unchained/pull/238/commits/143f663f3aa13c278ce57bef914db30cc1be0599